model_compression_toolkit/core/common/quantization/quantization_params_generation/qparams_computation.py

```diff
@@ -135,7 +135,7 @@ def calculate_quantization_params(graph: Graph,
                     attr_cfg.weights_channels_axis = (output_channels_axis, attr_cfg.weights_channels_axis[1])
                     attr_cfg.set_weights_quantization_param(weights_params)
 
-            if n.is_activation_quantization_enabled():
+            if n.is_activation_quantization_enabled() or n.is_fln_quantization():
```

Comment:
The condition here should be based on the fusing info, not the node candidate mode, similarly to the fix from PR #1462 
The same should apply for the condition in `set_activation_quantization_param` (the other change in this PR).